controllers/platformoperator_controller.go

```diff
@@ -118,31 +166,50 @@ func (r *PlatformOperatorReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		}
 		cb = append(cb, b)
 	}
-	if len(cb) == 0 {
-		log.Info("failed to find any candidate bundles")
-		return ctrl.Result{RequeueAfter: 5 * time.Second}, nil
-	}
 
 	latestBundle, err := cb.latest()
 	if err != nil || latestBundle == nil {
-		log.Info("failed to find the bundle with the highest semver")
+		message := "failed to find a valid bundle"
+		log.Info(message)
+		u.UpdateStatus(
+			updater.SetPhase(platformopenshiftiov1alpha1.PhaseFailing),
+			updater.EnsureCondition(metav1.Condition{
+				Type:    platformopenshiftiov1alpha1.TypeInstalled,
+				Status:  metav1.ConditionFalse,
+				Reason:  platformopenshiftiov1alpha1.ReasonNoValidBundles,
+				Message: fmt.Sprintf("%s with error: %v", message, err),
+			}),
+		)
 		return ctrl.Result{RequeueAfter: 5 * time.Second}, err
 	}
 
-	log.Info("candidate bundle", "bundle", latestBundle.CsvName, "version", latestBundle.Version)
+	log.Info("found a valid bundle", "bundle", latestBundle.CsvName, "version", latestBundle.Version)
 
-	// TODO: figure out what's the most appropriate field to parse by semver range. CsvName maybe if that's constantly present?
 	// TODO: what happens when the bundle failed?
 	// TODO: watch the bundle resource?
 	if err := r.ensureBundleInstance(ctx, po, latestBundle); err != nil {
```

Comment:
Going to think through logic that sets the correct status when a BundleInstance is in a failing install state. Right now it only fails on a unsuccessful application.